UI/TimeEntry/TimeEntryUno/TimeEntryUno.Wasm/TimeEntryUno.Wasm.csproj

```diff
@@ -35,18 +35,17 @@
     <None Include="LinkerConfig.xml" />
     <None Include="wwwroot\web.config" />
   </ItemGroup>
-  <ItemGroup>
-    <!-- Note that for WebAssembly version 1.1.1 of the console logger required -->
-    <PackageReference Include="IdentityModel" Version="5.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.1" />
-    <PackageReference Include="System.Text.Json" Version="6.0.0-preview.1.21102.12" />
-    <PackageReference Include="Uno.UI.WebAssembly" Version="3.7.4" />
-    <PackageReference Include="Uno.UI.RemoteControl" Version="3.7.4" Condition="'$(Configuration)'=='Debug'" />
+  <ItemGroup> 
+    <PackageReference Include="IdentityModel" Version="5.0.1" /> 
+    <PackageReference Include="System.Text.Json" Version="6.0.0-preview.1.21102.12" /> 
```

Comment:
It'd be best not to use preview bits. Was that required?

Reply:
It was already there, so i wasn't sure if would be better to upgrade. Done now!